src/wordpress/generator.py

```diff
@@ -277,6 +303,7 @@ def add_webmasters(self):
     def generate_mu_plugins(self):
         WPMuPluginConfig(self.wp_site, "epfl-functions.php").install()
         WPMuPluginConfig(self.wp_site, "EPFL-SC-infoscience.php").install()
+        WPMuPluginConfig(self.wp_site, "EPFL_custom_editor_menu.php").install()
```

Comment:
Est-ce qu'on pourrait pas simplement mettre ces noms de plugins dans une liste dans `settings.py` et itérer sur celle-ci au sein du code, ça serait plus propre non ?
Ou alors le tableau directement déclaré dans la fonction `generate_mu_plugins()` au moins.

Reply:
oui, c'est une bonne idée. D'autant plus qu'on a déjà une liste qui ferait l'affaire dans settings.py (pour l'instant nommée `WP_PLUGIN_CONFIG_CLASS_BY_NAME`)

Reply:
Euh, tu confonds avec un autre de mes commentaires là je crois non?... (celui-ci : https://github.com/epfl-idevelop/jahia2wp/pull/99#issuecomment-347225284 )

Parce que la variable `WP_PLUGIN_CONFIG_CLASS_BY_NAME` c'est pour faire un mapping entre un plugin et le nom de la classe à utiliser pour faire la configuration (ou alors quelque chose m'échappe).

Reply:
non non, je ne confond pas 😋 . l'idée était d'étendre son utilisation. Mais en regardant le code de plus près, je serai plutôt pour parti des fichiers yml que du fichier settings.py. Je serai même d'avis d'intégrer les classes de `WP_PLUGIN_CONFIG_CLASS_BY_NAME` dans le yml. Sinon, on a deux endroits différents pour caaractériser les plugins... pas top

Reply:
Effectivement, on pourrait étendre la structure du fichier YAML de configuration des plugins pour mettre le nom de la classe à utiliser pour effectuer la config.